src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -464,10 +453,10 @@ public List<uint256> GenerateStratisWithMiner(int blockCount)
         [Obsolete("Please use GenerateStratisWithMiner instead.")]
         public Block[] GenerateStratis(int blockCount, List<Transaction> passedTransactions = null, bool broadcast = true)
         {
-            FullNode fullNode = (this.runner as StratisBitcoinPowRunner).FullNode;
+            FullNode fullNode = ((StratisBitcoinPowRunner) this.runner).FullNode;
             BitcoinSecret dest = this.MinerSecret;
             var blocks = new List<Block>();
-            DateTimeOffset now = this.MockTime == null ? DateTimeOffset.UtcNow : this.MockTime.Value;
+            DateTimeOffset now = MockTime ?? DateTimeOffset.UtcNow;
```

Comment:
hmm strange, it's not asking you to write `MockTime.Value`?
Please add `this.` though.